python/lsst/validate/drp/matchreduce.py

```diff
@@ -212,29 +256,21 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
 
     for vId in dataIds:
 
-        if useJointCal:
-            try:
-                photoCalib = butler.get("jointcal_photoCalib", vId)
-            except (FitsError, dafPersist.NoResults) as e:
-                print(e)
-                print("Could not open photometric calibration for ", vId)
-                print("Skipping this dataId.")
-                continue
+        if doApplyExternalPhotoCalib:
             try:
-                wcs = butler.get("jointcal_wcs", vId)
+                photoCalib = butler.get('%s_photoCalib' % (externalPhotoCalibName), vId)
             except (FitsError, dafPersist.NoResults) as e:
                 print(e)
-                print("Could not open updated WCS for ", vId)
+                print("Could not open external photometric calibration for ", vId)
                 print("Skipping this dataId.")
                 continue
         else:
             try:
-                photoCalib = butler.get("calexp_photoCalib", vId)
+                photoCalib = butler.get('calexp_photoCalib', vId)
```

Comment:
Why change to single quotes here?

Reply:
Consistency with the other `butler.get` ...